framework/DataObjects/DataSet.py

```diff
@@ -1324,7 +1324,7 @@ def _fromXarrayDataset(self,dataset):
         self.raiseAnError(KeyError,'Dimensions of variable',var,'from "source"', ",".join(providedDims),
                 'is not consistent with the required dimensions for data object "',
                 self.name.strip(),'":',",".join(requiredDims))
-    self._orderedVars = self.vars + self.indexes
+    self._orderedVars = self.vars
```

Comment:
?

Reply:
Based on my understanding,  ```_orderedVars``` is used to store all the variables, e.g. inputs and outputs, not indexes.

Reply:
I think including the indexes was meant to provide consistent mapping to the underlying CND-array, but it's been a while since I checked. I suppose if this didn't crash any tests then we haven't really relied on the indices being there yet ...